app/server/me_api/misc.py

```diff
@@ -282,4 +286,106 @@ def post(self):
             'data': create_payout_account_response
         }
 
-        return make_response(jsonify(response_object)), 201
\ No newline at end of file
+        return make_response(jsonify(response_object)), 201
+
+
+class PoliPaymentsAPI(MethodView):
+    @requires_auth
+    def get(self, reference):
+        reference = request.args.get('reference')
+        if reference:
+
+            fiat_ramp = FiatRamp.query.filter_by(payment_reference=reference).first()
+            if fiat_ramp is None:
+                return make_response(jsonify({'message': 'Could not find payment for reference {}'.format(reference)})), 400
+
+            poli_link = fiat_ramp.payment_metadata['poli_link']
+            try:
+                get_poli_link_status_response = get_poli_link_status(poli_link=poli_link.split('/')[-1])
```

Comment:
again here, a bit confusing where poli_link is different from poli_link. seems we only ever use that last part, so maybe we should just store that as poli_link? or name the last section "poli_link_id"?

Reply:
Yes, will update